app/views/components/paragraph.scala.html

```diff
@@ -0,0 +1,18 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(text: String, id: Option[String] = None)
```

Comment:
Maybe empty string as null object pattern ?

Reply:
I'd rather avoid creating this component if it is empty. Currently we have no such use case.

Reply:
My bad - i have in mind `id` part of it - if it is empty is should nit be rendered - empty `id` tag is meaningless and having `Option[String]` give option to put empty string as `id`